feat: add support for windows

lswest opened the pull request:
I just wanted to use this on a project under Windows today, and discovered that it wasn't finding any matches (so all files were being returned as not imported).

I did some debugging in my `node_modules` folder and discovered that the comparison (`.has(x)`) ends up comparing a path that looks like: `C:\\Repositories\\file.tsx` with `C:/Repositories/files.tsx`, so I changed the lines to `.has(x.replace(/\\/g, '/'))` and started seeing results like I was expecting.

I'm not sure what you'd prefer - running the replace inline during the `.has()`, or if we want to do it in index.ts on line 135 (where you resolve the glob `'**/*'`). As such, feel free to give me some feedback as to how you'd like it done.

I haven't validated if there are any other Windows issues yet.

smeijer commented:
Hi! Thanks for the PR! 

Regarding the prefered solution, I think it's best to 'normalize' the slashes as soon as possible, so that they are kept in a uniform format in memory, wherever we'd access the file list.

[`index.ts#135`](https://github.com/smeijer/unimported/blob/d2ff96055afbf9a48440d7484139bae164906672/src/index.ts#L135-L138) sounds good to me. Can you verify if that works, and update the PR?

https://github.com/smeijer/unimported/blob/d2ff96055afbf9a48440d7484139bae164906672/src/index.ts#L135-L138

lswest commented:
@smeijer You're welcome! I've implemented the changes, tested it under both Linux and Windows, and fixed some issues with the tests that wouldn't run on Windows.

I'll also be opening a separate PR for the invalid JSON issue in a second.

The issues in the tests:
- The path separators had to be replaced in the tests as well
- You set LC_ALL in the help tests in the *nix way (`LC_ALL='en'`), so I updated the test to use a command that would work in Windows based on `process.platform`.

smeijer commented:
Thanks! Merged, and published as `1.8.0`.
